.gitignore

```diff
@@ -103,7 +102,6 @@ gcp/create-gke/addons/openebs/terraform.tfstate.backup
 gcp/create-gke/addons/openebs/tflogs/
```

Comment:
what are you thoughts about doing this instead?
`gcp/*tflogs
gcp/*/*tflogs
gcp/*/*/*tflogs`

Reply:
That makes sense, since you have multiple sub-directories with other log files too.

Reply:
I think we can do one better:
```
*.log
**/*logs
```

This will match all `*.log` files and any `*logs` directories.